yamlparse-applicative/src/YamlParse/Applicative/Pretty.hs

```diff
@@ -38,8 +39,23 @@ prettyParserDoc = prettySchema . explainParser
 prettySchema :: Schema -> Text
 prettySchema = renderStrict . layoutPretty defaultLayoutOptions . schemaDoc
 
+-- | Render a schema as pretty and colorized text.
+--
+-- This is meant for humans.
+-- The output may look like YAML but it is not.
+prettyColorizedSchema :: Schema -> Text
+prettyColorizedSchema = renderSimplyDecorated id startColor resetColor . layoutPretty defaultLayoutOptions . schemaDoc
+
+startColor :: Color -> Text
```

Comment:
You can put `startColor` and `resetColor` in a `where` clause of `prettyColorisedSchema` because you don't need it outside of that function.

Reply:
ah right, I keep forgetting that

Reply:
done